src/uglify/cache.js

```diff
@@ -0,0 +1,23 @@
+import flatCache from 'flat-cache';
+
+const cacheId = 'uglifyjs-webpack-plugin';
+
+export default class Cache {
```

Comment:
Why do we need a class here ?

```js
const cache = {
  get (dir, key) {},
  put (dir, key, data) {},
  list (dir) {},
  clear (dir) {}
}

export default cache

Reply:
We don't necessarily need a class, no. Something with a similar api was needed for the Jest spies but that's it. I'll change it to an object if it's preferable.

Reply:
Bear in mind that changing it from `Cache` to `cache` necessitated a few other renames to prevent shadowed variables.

Reply:
Loading the ?whole cache `flatCache.load(id, dir)` on every method call seems odd or I'm missing something ?